### ISSUE TITLE
Error loading homebridge plugin

### ISSUE THREAD

**Issue by avocade** [stenehall/homebridge-ikea#1]
Not sure if this is me or homebridge or the plugin (using latest homebridge globally installed).

Installed the plugin with `npm install -g homebridge-ikea`.

```bash
 $ homebridge
[4/10/2017, 2:15:29 PM] ====================
[4/10/2017, 2:15:29 PM] ERROR LOADING PLUGIN homebridge-ikea:
[4/10/2017, 2:15:29 PM] /usr/local/lib/node_modules/homebridge-ikea/index.js:25
  accessories: async function(callback) {
                     ^^^^^^^^
SyntaxError: Unexpected token function
    at Object.exports.runInThisContext (vm.js:78:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Plugin.load (/usr/local/lib/node_modules/homebridge/lib/plugin.js:65:22)
    at Server.<anonymous> (/usr/local/lib/node_modules/homebridge/lib/server.js:132:14)
[4/10/2017, 2:15:29 PM] ====================
```

**Comment by jannemann** [comment]
I got a similar error with node 4 and upgraded to 7. This resolved my issues.

**Comment by avocade** [comment]
Solved it, thanks @jannemann.

tl;dr if you're on Mac run `brew upgrade nodejs && npm install -g homebridge-ikea`.

**Comment by stenehall** [comment]
Correct, you need something like Node 7.6 where async/await was introduced. I added that dependency as https://github.com/stenehall/homebridge-ikea/blob/master/package.json#L24 but I don't think node really cares about that.
